frontend/nc-news-app-jg/src/Components/ArticlesList.jsx

```diff
@@ -13,16 +14,20 @@ export default function ArticlesList() {
 	return (
 		<div className="articlesList">
 			<section>
-				<ul>
-					List of articles should display here
+				<ul className="article-list-ul">
 					{articles.map((article) => {
-						return ( 
-
-            <li key={article.article_id}>
-             <p>{article.title} </p>
-              {/* here will be a link to go to single Article card which will get clicked article by its ID */}
-            </li>)
-           
+						return (
+							<li className="singleArticle" key={article.article_id}>
+								<h4>{article.title} </h4>
+								<p>Topic: {article.topic}</p>
+								<Link
+									to={`/articles/${article.article_id}`}
+									className="linkTopicCard"
+								>
+									<button>Go to article</button>
+								</Link>
```

Comment:
as discussed r.e accessibility consider just using links for routing and buttons to trigger an action :)